src/Engine/Component/GeometryComponent.cpp

```diff
@@ -105,170 +101,170 @@ void FancyMeshComponent::handleMeshLoading( const Ra::Asset::GeometryData* data
 
     if ( data->hasTangents() )
     {
-        displayMesh->addData( Ra::Engine::Mesh::VERTEX_TANGENT, data->getTangents() );
+        displayMesh->addData( Mesh::VERTEX_TANGENT, data->getTangents() );
     }
 
     if ( data->hasBiTangents() )
     {
-        displayMesh->addData( Ra::Engine::Mesh::VERTEX_BITANGENT, data->getBiTangents() );
+        displayMesh->addData( Mesh::VERTEX_BITANGENT, data->getBiTangents() );
     }
 
     if ( data->hasTextureCoordinates() )
     {
-        displayMesh->addData( Ra::Engine::Mesh::VERTEX_TEXCOORD, data->getTexCoords() );
+        displayMesh->addData( Mesh::VERTEX_TEXCOORD, data->getTexCoords() );
     }
 
     if ( data->hasColors() )
     {
-        displayMesh->addData( Ra::Engine::Mesh::VERTEX_COLOR, data->getColors() );
+        displayMesh->addData( Mesh::VERTEX_COLOR, data->getColors() );
     }
 
     // FIXME(Charly): Should not weights be part of the geometry ?
```

Comment:
Why Charly is still there ??? This FIXME is very old ... (from the begining of the Radium dev).